test/index.test.js

```diff
@@ -32,299 +35,386 @@ Object.defineProperty(loaderContextMock, "options", {
+                        .then(() => {
+                            const expectedCss = readCss("scss", "imports");
+                            const a = readBundle("bundle.multiple-compilations.a.js");
+                            const b = readBundle("bundle.multiple-compilations.b.js");
+                            const c = readBundle("bundle.multiple-compilations.c.js");
+                            const d = readBundle("bundle.multiple-compilations.d.js");
+                            const e = readBundle("bundle.multiple-compilations.e.js");
+                            const f = readBundle("bundle.multiple-compilations.f.js");
+                            const g = readBundle("bundle.multiple-compilations.g.js");
+                            const h = readBundle("bundle.multiple-compilations.h.js");
 
-                    a.should.equal(expectedCss);
-                    b.should.equal(expectedCss);
-                    c.should.equal(expectedCss);
-                    d.should.equal(expectedCss);
-                    e.should.equal(expectedCss);
-                    f.should.equal(expectedCss);
-                    g.should.equal(expectedCss);
-                    h.should.equal(expectedCss);
-                })
-        );
-    });
-    describe("source maps", () => {
-        function buildWithSourceMaps() {
-            return new Promise((resolve, reject) => {
-                runWebpack({
-                    entry: path.join(__dirname, "scss", "imports.scss"),
-                    output: {
-                        filename: "bundle.source-maps.js"
-                    },
-                    devtool: "source-map",
-                    module: {
-                        rules: [{
-                            test: /\.scss$/,
-                            use: [
-                                { loader: testLoader.filename },
-                                {
-                                    loader: pathToSassLoader, options: {
-                                        sourceMap: true
-                                    }
-                                }
-                            ]
-                        }]
-                    }
-                }, err => err ? reject(err) : resolve());
+                            a.should.equal(expectedCss);
+                            b.should.equal(expectedCss);
+                            c.should.equal(expectedCss);
+                            d.should.equal(expectedCss);
+                            e.should.equal(expectedCss);
+                            f.should.equal(expectedCss);
+                            g.should.equal(expectedCss);
+                            h.should.equal(expectedCss);
+                        })
+                );
             });
-        }
+            describe("source maps", () => {
+                function buildWithSourceMaps() {
+                    return new Promise((resolve, reject) => {
+                        webpack({
```

Comment:
Why can't you re-use `runWebpack` here?

Reply:
This uses different values for `module.rules`, which `webpack-merge` can't merge in (it would just try to add rules to the list rather than replacing it entirely).